src/runtimeHelpers.ts

```diff
@@ -261,7 +261,7 @@ export function unPartitionResults<T>(
  */
 export function resultsDictToList<V>(
     /** A dictionary of results */
-    response: { [key in string | number]: V },
+    response: { [key: string]: V },
```

Comment:
hmm what about `number` tho? see the example function usage in the docstring

Reply:
Oh I see this is never the case according to the spec

neato! let's maybe include a link explaining this as a comment somewhere?

(if i'm understanding it correctly, even though the json response from the network might send down numbers for keys, it will always be turned into a string? that's worth calling out)